index.html

```diff
@@ -267,20 +267,23 @@
       <li>Create and expose a local <a>Thing</a> based on a <a>Thing Description</a>.</li>
       <li>Programmatically create and expose a local <a>Thing</a>. This may include the following operations:
         <ul>
+          <li>Register the Thing.</li>
+          <li>Unregister the Thing.</li>
+          <li>Start the exposed <a>Thing</a> in order to process external requests.</li>
+          <li>Stop the exposed <a>Thing</a>.
           <li>Add a property definition to the <a>Thing</a>.</li>
           <li>Add an event definition to the <a>Thing</a>.</li>
           <li>Add an action definition to the <a>Thing</a>.</li>
-          <li>Attach semantic information to an Action</li>
-          <li>Attach semantic information to a Property</li>
-          <li>Attach semantic information to an Event</li>
-          <li>Start the exposed <a>Thing</a> in order to process external requests.</li>
-          <li>Modify the handling of an external request to retrieve the <a>Thing Description</a>.</li>
-          <li>React on a property change</li>
+          <li>Register an action triggered by external retrieve requests on the <a>Thing Description</a>.</li>
```

Comment:
to avoid confusion with the interacion pattern "action" (of a thing) , could we say "a callback" or "logic"?